File: Assignment Programs/Assignment42/program2.c
#include<stdio.h>
#include<io.h>
#include<fcntl.h>

int CountSmall(char FName[])
{
	int fd = 0, iRet = 0, i = 0, iCnt = 0;
	char Buffer[10];
	fd = open(FName,O_RDONLY);
	
	while((iRet = read(fd,Buffer,sizeof(Buffer))) != 0)
	{
		for(i = 0;i<iRet;i++)
		{
			if((Buffer[i] >= 'a') && (Buffer[i] <= 'z'))
			{
				iCnt++;
			}
		}
	}
	close(fd);
	return iCnt;
}

int main()
{
	char FileName[30];
	int iRet = 0;
	printf("Enter file name\n");
	scanf("%s",FileName);	
	iRet = CountSmall(FileName);
	printf("Number of Small characters are %d",iRet);
	return 0;
}

File: Assignment Programs/Assignment31/program2.java
import java.util.*;

class StringX
{
	public String strI;
	
	public void Accept()
	{
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter string :");
		strI = sobj.nextLine();
	}
}

class StringDemo extends StringX
{
	public int CountSmall(String str)
	{
		int iCnt = 0, i = 0;
		char Arr[] = str.toCharArray();
		
		for(i = 0; i<Arr.length; i++)
		{
			if((Arr[i] >= 'a') && (Arr[i] <= 'z'))
			{
				iCnt++;
			}
		}
		return iCnt;
	}
}

class program2
{
	public static void main(String args[])
	{
		int iRet = 0;
		StringDemo sobj = new StringDemo();
		sobj.Accept();
		iRet = sobj.CountSmall(sobj.strI);
		System.out.println("Number of small characters in given string are : "+iRet);
	}
}